packages/daemon/src/service/pipeline.ts

```diff
@@ -276,14 +277,13 @@ export class PipelineService {
     }
   }
 
-  stopJob(id: string): boolean {
+  stopJob(id: string): void {
     const runnable = runnableMap[id];
     if (runnable) {
       runnable.destroy();
       delete runnableMap[id];
-      return true;
     } else {
-      return false;
+      this.ctx.throw('no runnable found');
```

Comment:
Throw with an error, please

Reply:
In service and controller, the error should be throw by the context object, the error will bring the http status.